corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -49,14 +49,15 @@ hqDefine("cloudcare/js/formplayer/menus/api", [
 
             $.when(AppsAPI.getAppEntities()).done(function (appCollection) {
                 const app = appCollection.find(function (app) {
+                    const currCopyOf = app.get('copy_of');
```

Comment:
nit: I think a name that denotes it's the primary app id of the application would be more descriptive. I think that's always going to be the case? So maybe something like `primaryAppId`?

Reply:
I could see using "primary" in the var name. I'm using "curr" to denote that we are looping over apps in `appCollection` and `currCopyOf` is the primary app id of the app we are currently dealing with in the loop. Maybe `currPrimaryAppId` would be more clear.

Reply:
that sounds good to me